libraries/stdlib/samples/test/samples/collections/sort.kt

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package samples.collections
+
+import samples.*
+import java.time.LocalDateTime
+
+@RunWith(Enclosed::class)
+class Sort {
```

Comment:
Please use another placement schema, similar to other collection and array samples.
Collection and List samples should go into `collections.kt`, and `Sorting` nested class can be created there, so its full name would be `samples.collections.Collections.Sorting`. Ditto for the array sorting samples.